profiles/common/modules/custom/tmgmt_poetry/src/tmgmt_poetry.behat.inc

```diff
@@ -228,6 +228,16 @@ class TmgmtPoetrySubContext extends RawDrupalContext implements DrupalSubContext
   }
 
   /**
+   * Generate a relative date for job deadline.
+   *
+   * @When I fill in :arg1 with a relative date of :arg2 days
+   */
+  public function iFillInWithArelativeDateOfDays($field, $number) {
+    $relative_date = date('d/m/Y', strtotime($number));
+    $this->getSession()->getPage()->fillField($field, $relative_date);
```

Comment:
Could insert here a check:

// Retrieve the specified field.
    if (!$field = $this->getSession()->getPage()->findField($field)) {
      throw new ExpectationException("Field '$field' not found.", $this->getSession());
    }